### PR TITLE
improvement/dark-styles-tweaks

### PR DESCRIPTION
**Pull request by dabeeeenster**
From 
image

to 
image

### CODE REVIEW

File: frontend/common/project.js [hunk 1]
```diff
@@ -1,18 +1,15 @@
 module.exports = global.Project = {
-    api: 'https://api.flagsmith.com/api/v1/',
```
Comment (matthewelwell) [inline frontend/common/project.js]: Can you revert the changes to this file please?

**Review by kyle-ssg — APPROVED** [review]

**Comment by dabeeeenster** [comment]
👀
